src/pgsql/bin/entrypoint.sh

```diff
@@ -10,10 +10,14 @@ echo '>>> STARTING SSH (if required)...'
 source /home/postgres/.ssh/entrypoint.sh
 
 echo '>>> STARTING POSTGRES...'
-/usr/local/bin/cluster/postgres/entrypoint.sh & wait ${!} || echo ">>> Foreground processes returned code: '$?'"
+/usr/local/bin/cluster/postgres/entrypoint.sh &
+wait ${!}
+code=$?
```

Comment:
All vars are in capitals in the project ;)

Reply:
fixed